src/other/utils.jl

```diff
@@ -128,3 +128,14 @@ function tforeach(f, x::AbstractArray; basesize::Integer)
     end
     return
 end
+
+function _nt_like_hash(v, h::UInt)
```

Comment:
Is this faster than just doing `hash(NamedTuple(v), h)`?

Reply:
doing `NamedTuple(v)` can have a prohibitive compilation cost for wide data frames. Also it is faster. Here are timings for 5-column data frame:
```
julia> @benchmark _nt_like_hash($df[1, :], UInt(0))
BenchmarkTools.Trial:
  memory estimate:  368 bytes
  allocs estimate:  14
  --------------
  minimum time:     589.655 ns (0.00% GC)
  median time:      598.276 ns (0.00% GC)
  mean time:        654.433 ns (1.49% GC)
  maximum time:     9.713 μs (89.57% GC)
  --------------
  samples:          10000
  evals/sample:     174

julia> @benchmark hash(NamedTuple($df[1, :]))
BenchmarkTools.Trial:
  memory estimate:  416 bytes
  allocs estimate:  11
  --------------
  minimum time:     1.310 μs (0.00% GC)
  median time:      1.450 μs (0.00% GC)
  mean time:        1.537 μs (0.00% GC)
  maximum time:     5.290 μs (0.00% GC)
  --------------
  samples:          10000
  evals/sample:     10
```